README.md

```diff
@@ -43,14 +44,16 @@ Artefacts can be specified as flags or as environment variables with the followi
 | `--docker-images` | docker-image docker-image | A white-space delimited set of docker images | `mysql alpine` |
 | `--image-vars` | `"MYSQL_IMAGE ANOTHER_IMAGE"` | A white-space delimited set of image variable names | Given:</br>`export MYSQL_IMAGE=mysql:v5.0`</br>`export ALPINE_IMAGE=alpine` </br> Use: </br>`"MYSQL_IMAGE ALPINE_IMAGE"`|
 | `--web-files` | url,filename,sha256[,true/false] | A white-space separated list of CSV's in the following format: </br></br>`url` is where to download from</br></br> `filename` is the name to save locally</br></br> `sha256` is the expected checksum</br></br>The optional last parameter specifies if the file should have executable permissions | `https://bit.ly/2ySXztI,kd,2f7...,true https://bit.ly/abc.iso,my.iso,abc...` |
-
+| `--docker-username` | `usernmame` | A valid docker registry user-name see # | `bob` |
```

Comment:
`see #`?

Reply:
oops - missed that - will make a README fix up later new issue :)